sdk/src/main/scala/com/horizen/account/state/ForgerStakeMsgProcessor.scala

```diff
@@ -474,17 +474,33 @@ object ForgerStakeMsgProcessor {
 case class AccountForgingStakeInfo(
                                     stakeId: Array[Byte],
                                     forgerStakeData: ForgerStakeData)
-  extends BytesSerializable  {
+  extends BytesSerializable with ABIEncodable {
 
   override type M = AccountForgingStakeInfo
 
   override def serializer: ScorexSerializer[AccountForgingStakeInfo] = AccountForgingStakeInfoSerializer
 
   override def toString: String = "%s(stakeId: %s, forgerStakeData: %s)"
     .format(this.getClass.toString, BytesUtils.toHexString(stakeId), forgerStakeData)
+
+  private[horizen] def asABIType(): StaticStruct = {
+
+    val forgerPublicKeysParams: util.List[Type[_]] = forgerStakeData.forgerPublicKeys.asABIType().getValue.asInstanceOf[util.List[Type[_]]]
```

Comment:
I added the .asInstanceOf[util.List[Type[_]] because maven didn't compile. I have removed it and now it compiles but I don't understand what is different now...